service/github/github.go

```diff
@@ -300,12 +301,17 @@ func (g *PullRequest) setPostedComment(ctx context.Context) error {
 
 // Diff returns a diff of PullRequest.
 func (g *PullRequest) Diff(ctx context.Context) ([]byte, error) {
-	opt := github.RawOptions{Type: github.Diff}
-	d, _, err := g.cli.PullRequests.GetRaw(ctx, g.owner, g.repo, g.pr, opt)
```

Comment:
Can we keep this code as-is and use git-diff solution only when API request failed for backward compatibility?

Reply:
I apply it: https://github.com/reviewdog/reviewdog/compare/3c496f50759521c76f0d43f38d0eefaebf8819e9...e67af34299d57ea9b857d71cfccdcea2cf0cb5dc